hs_core/hydroshare/social.py

```diff
@@ -1,189 +0,0 @@
-        new_comment.save()
-
-    return new_comment
-
-
-def endorse_comment(comment_id, resource_short_id, user, endorse=True):
-    """
-    Adds a +1 to the comment from the user.  Checks first to make sure the user has not +1'd previously.
-
-    :param comment_id: The comment ID (primary key) of the comment to be endorsed
-    :param resource_short_id: short_id of the resource (for user access authorization purposes)
-    :param user: The user endorsing the comment. user id, user name or user instance
-    :param endorse: True for a +1.  False to remove any previous endorsement.
-    :return: The rating instance (Rating)
-    """
-
-    resource = utils.get_resource_by_shortkey(resource_short_id)
-    user = utils.user_from_id(user)
-    comment_to_endorse = ThreadedComment.objects.get(pk=comment_id)
-
-    # check the comment belongs to the specified resource
-    if comment_to_endorse.content_object != resource:
-        raise ValueError("Comment does not belong to the specified resource")
-
-    # first, make sure the user hasn't already added a rating to this comment.
-    comment_type = ContentType.objects.get_for_model(ThreadedComment)
-    rating = Rating.objects.filter(content_type=comment_type, object_pk=comment_to_endorse.id, user=user).first()
-
-    # if user has already endorsed and endorse is false, delete the Rating object
-    if rating and not endorse:
-        rating.delete()
-    # user can't endorse the same comment twice -just return the user's existing rating object
-    elif rating and endorse:
-        return rating
-    # if user has not endorsed this comment before and endorse is true then create a new rating object
-    elif not rating and endorse:
-        rating = Rating.objects.create(content_object=comment_to_endorse, user=user, value=1)
-        return rating
-
-
-def get_comments(resource_short_id):
-    """
-    Get the list of comment for a resource identified by pid.
-
-    :param resource_short_id: short_id of the resource (for user access authorization purposes)
-
-    """
-
-    # get resource instance
-    resource = utils.get_resource_by_shortkey(resource_short_id)
-
-    # get all comments objects of the resource
-    comments = ThreadedComment.objects.filter(object_pk=resource.pk)
-
-    return comments
-
-
-def get_endorsements(for_object):
-    """
-    Get the list of rating objects for a resource or a comment object.
-
-    :param for_object an object of a resource or a comment
-
-    """
-
-    if isinstance(for_object, BaseResource):
-        resource_type = ContentType.objects.get_for_model(BaseResource)
-        endorsements = Rating.objects.filter(content_type=resource_type, object_pk=for_object.id)
-
-    elif isinstance(for_object, ThreadedComment):
-        comment_type = ContentType.objects.get_for_model(ThreadedComment)
-        endorsements = Rating.objects.filter(content_type=comment_type, object_pk=for_object.id)
-
-    return endorsements
```

Comment:
@pkdash yes, this PR work is not related to this file, so this file removal is just a by-product of this PR. As long as you are OK with it, I think it is OK to remove unused code in a unrelated PR rather than waiting to do bulk removal all together.